pkg/apis/k0s/v1beta1/api.go

```diff
@@ -37,6 +37,10 @@ type APISpec struct {
 	// Address on which to connect to the API server.
 	Address string `json:"address,omitempty"`
 
+	// Whether to only bind to the IP given by the address option.
+	// +optional
+	OnlyToBindAddress bool `json:"onlyBindToAddress,omitempty"`
```

Comment:
```suggestion
	OnlyBindToAddress bool `json:"onlyBindToAddress,omitempty"`
```

IMO worth to keep the variable name and json key in sync

Reply:
ops, yeah I'll fix that in the evening.